src/plugins/menu/_noscript.scss

```diff
@@ -6,26 +6,33 @@
 /*
  * Menu Sass (noscript)
  */
-#wb-glb-mn {
-	display: none !important;
-}
 
-#wb-srch {
-	@extend %noscript-display-block-important;
-}
+@media screen {
+	%menu-noscript-screen-display-block-important {
+			display: block !important;
+	}
 
-#wb-sm {
-	@extend %noscript-display-block-important;
+	#wb-glb-mn {
```

Comment:
Is this selector should be inside or outside the ```@media screen``` query? Because in your previous code revision it was outside I think.